tools/cli/src/cargo_manifest.rs

```diff
@@ -1,30 +1,27 @@
-use cargo_toml::{Dependency, Error as CargoTomlError, Manifest};
+use cargo_toml::{Error as CargoTomlError, Manifest};
 use toml::de::Error as TomlError;
 use semver::Version;
 use thiserror::Error as ThisError;
 
 use std::path::Path;
-use std::str::FromStr;
-use crate::cargo_manifest::ManifestError::CannotProcessManifest;
 
 const SKD_CRATE_NAME: &str = "marine-rs-sdk";
+const LOCKFILE_NAME: &str = "Cargo.lock";
 
 #[derive(Debug, ThisError)]
 pub enum ManifestError {
+    #[error("Non-marine wasm, is to be skipped")]
```

Comment:
Suggest writing here that there is no `marine-rs-sdk` used in Wasm module compilation